calibrations/mbd/prorun_mbdcal.sh

```
@@ -0,0 +1,277 @@
+# Pass 2 calibrations mip fits
+#fname=$(ls -tr DST_UNCALMBD*.root | tail -1)
+#echo Waveform processing done
+#ls -ltr
+
+#fname=$(head -1 ${inputs})
+
+echo "###############################################################################################################"
+echo "Running pass2.0 calibration"
+#./cups.py -r ${runnumber} -s ${segment} -d ${outbase} message "Running PASS 2.0 calibration"
+pass=0
+echo root.exe -q -b Fun4All_MbdCalReco.C\(\"${inputs}\",${pass},${nevents},\"${dbtag}\"\)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)